barcode_action/static/src/js/action_barcode_form.js

```diff
@@ -1,23 +1,24 @@
-odoo.define('barcode_action.form', function (require) {
+odoo.define("barcode_action.form", function(require) {
     "use strict";
 
-    var FormController = require('web.FormController');
+    var FormController = require("web.FormController");
 
     FormController.include({
-        _barcodeHandleAction: function (barcode, activeBarcode) {
+        _barcodeHandleAction: function(barcode, activeBarcode) {
```

Comment:
```suggestion
        _barcodeHandleAction: function(barcode) {
```

This should make the linter happy again.